tools/abel/validators/README.md

```diff
@@ -0,0 +1,197 @@
+# ABEL Spreadsheet
+
+### Overview
+The ABEL Spreadsheet serves as a user-friendly interface for ABEL and allows users to create a
+[Building Configuration file](../../../ontology/docs/building_config.md) from scratch.
+
+***Note:*** ABEL can ingest Building Config files using the [initialize
+operation](../../../ontology/docs/building_config.md#initialize) but it does not
+currently support [update
+operations](../../../ontology/docs/building_config.md#update)
+
+### Generating GUIDs
+
+There are two use cases for ABEL,
+[brownfield](../../../ontology/docs/building_config.md#update) and
+[greenfield](../../../ontology/docs/building_config.md#initialize). Currently ABEL
+only supports greenfield buildings or Building Configs under the [Initialize
+operation](../../../ontology/docs/building_config.md#initialize)
+
+#### Greenfield building
+
+A [greenfield building](../../../ontology/docs/building_config.md#initialize) 
+is one that has yet to be onboarded, and ABEL is used to 
+create a Building Config from scratch to model this building. In
+this case **GUIDs do not need to be provided except for when there are
+duplicate entity codes.** Then and only then will the user need to provide
+GUIDs for the entities with duplicate codes in every one of the ABEL
+spreadsheet tables.
+
+## Google Sheets template
+
+The [ABEL Spreadsheet Template](https://docs.google.com/spreadsheets/d/1tcLjFnHiXUT-xh5C1hRKiUVaUH_CzgSI8zFQ_B8q7vs/copy#gid=980240783) has a strict format that must be adhered to.
+
+Below is a definition of each table and column in the ABEL spreadsheet:
+- [Site](#site)
+- [Entities](#entities)
+- [Entity Fields](#entity-fields)
+- [States](#states)
+- [Connections](#connections)
+
+### Site
+
+The `Site` tab holds all information about a project site or building.
+
+`Building Code` *string* **required**
+
+A [human-readable code](../../../ontology/docs/building_config.md#identifiers) for a building. e.g. `US-SVL-TC2`
```

Comment:
Must use fake name for site on any open-source project documents. Use `US-SEA-BLDG1` or something similar.